api/gdpr.py

```diff
@@ -0,0 +1,165 @@
+                    res.billing_first_name,
+                    res.billing_last_name,
+                    res.billing_email,
+                    res.billing_phone,
+                    res.billing_address_zip,
+                    res.billing_address_city,
+                    res.billing_address_street,
+                    res.reservee_id,
+                    res.reservee_organisation_name,
+                    res.free_of_charge_reason,
+                    res.cancel_details,
+                ]
+            )
+        return reservations
+
+    @property
+    def applications(self) -> List:
+        applications = []
+        for app in self.user.application_set.all():
+            applications.append(app.additional_information)
+            events = []
+            for e in app.application_events.all():
+                events.append(e.name)
+                events.append(e.name_fi)
+                events.append(e.name_en)
+                events.append(e.name_sv)
+            applications.append({"events": events})
+            if app.contact_person:
+                applications.append(
+                    {
+                        "contact_person": [
+                            app.contact_person.first_name,
+                            app.contact_person.last_name,
+                            app.contact_person.email,
+                            app.contact_person.phone_number,
+                        ]
+                    }
+                )
+
+            if app.organisation:
+                applications.append(
+                    {
+                        "organisation": [
+                            app.organisation.name,
+                            app.organisation.identifier,
+                            app.organisation.email,
+                            app.organisation.core_business,
+                            app.organisation.core_business_fi,
+                            app.organisation.core_business_en,
+                            app.organisation.core_business_sv,
+                        ]
+                    }
+                )
+            if app.organisation and app.organisation.address:
+                applications.append(
+                    {
+                        "organisation_address": [
+                            app.organisation.address.post_code,
+                            app.organisation.address.street_address,
+                            app.organisation.address.street_address_fi,
+                            app.organisation.address.street_address_en,
+                            app.organisation.address.street_address_sv,
+                            app.organisation.address.city,
+                            app.organisation.address.city_fi,
+                            app.organisation.address.city_en,
+                            app.organisation.address.city_sv,
+                        ]
+                    }
+                )
```

Comment:
Empty line before next `if` here as well could help.

These are very much up to preference and some might hate it that there is much whitespace in code.

However, even though there are a few bytes more in the file, it does make readability more enjoyable.